src/mono/wasm/runtime-test.js

```diff
@@ -130,6 +130,9 @@ while (true) {
 	} else if (args [0] == "--disable-on-demand-gc") {
 		enable_gc = false;
 		args = args.slice (1);
+	} else if (args [0] == "--enable-gc") {
```

Comment:
this was removed in favor of --disable-on-demand-gc above since it is now the default.  Was some other code calling it?

Reply:
some tests were passing it as an argument but i think that was a local thing